src/stubs.ts

```diff
@@ -233,6 +233,10 @@ export const createFigma = (config: TConfig): PluginAPI => {
   }
 
   class LayoutMixinStub implements LayoutMixin {
+    layoutGrow: number;
+    rescale(scale: number): void {
+      return;
```

Comment:
Maybe we also need to do scaling:
```ts
this.width = this.width * scale;
this.height = this.height * scale;
```

Reply:
Applied your suggestion along with error handling for rescale factor to be >= 0.01